sys/kern/subr_uio.c

```diff
@@ -554,7 +554,7 @@ fuword32(volatile const void * __capability addr)
 	return (rv == -1 ? -1 : val);
 }
 
-#ifdef _LP64
+#if defined(_LP64) || __has_feature(capabilities)
```

Comment:
`__SIZEOF_SIZE_T__ == 8` (not long as you'd want this for LLP64 too, not that we _really_ care about that)